public/components/incontext_insight/generate_popover_body.tsx

```diff
@@ -251,6 +312,13 @@ export const GeneratePopoverBody: React.FC<{
     <>
       {renderInnerTitle()}
       {renderContent()}
+      {displayDiscoverButton && (
+        <EuiButton onClick={handleNavigateToDiscover}>
+          {i18n.translate('assistantDashboards.incontextInsight.discover', {
+            defaultMessage: 'Discover details',
+          })}
+        </EuiButton>
```

Comment:
this is not same as mock 
![image](https://github.com/user-attachments/assets/3b11b2c3-fd7f-4f9f-b181-a82267e71081)

Reply:
This icon is not still introduced in OUI, and we don't need to rush UI to completely same as UI is still in progress of improvement.